### DISCUSSION TITLE
Requested Items can be sold by the merchant NPC

### DISCUSSION THREAD

**Discussion by ubertpendragon** [FakeFishGames/Barotrauma#14374] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

Merchants can sell their requested items as they are not exclusive.

This gives very little room to change an outpost's store values because it's possible that you can buy the requested goods then sell them for easy profit.

Requested Items should be removed from the shop list?
This wouldn't prevent other merchants from selling the item though

### Version

v1.5.9.1 (Summer Update Hotfix 2)

**Comment by Regalis11** [comment]
This has been addressed since the report was created, looks like we just forgot to close the ticket. :)
